### PR TITLE
Weaver & Woodcutter Hotfix

### PR DESCRIPTION
**Pull request by Max-023**
## About The Pull Request
Apparently weaver and woodcutter were displayed as combat classes, so this PR should fix that.

## Why It's Good For The Game
They aren't combat classes lol.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by Max-023** [comment]
> merge conflict

Revolved!

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/weaver.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
-/datum/advclass/weaver
+/datum/advclass/pilgrim/weaver
 	name = "Weaver"
-	allowed_sexes = list("Male", "female")
+	allowed_sexes = list("male", "female")
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/weaver.dm:3]:
```suggestion
	allowed_sexes = list(MALE, FEMALE)
```
For further reference for any coders, capitalizing the words with no quotation marks is better for standardization.

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/weaver.dm [hunk 2]
```diff
@@ -13,6 +13,7 @@
 	ispilgrim = FALSE
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/weaver.dm:13]:
```suggestion
	ispilgrim = TRUE
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

**Review by ThePainkiller — APPROVED** [review]
All good now.

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-	allowed_sexes = list(MALE)
-	allowed_races = list(
-		"Humen",
-		"Elf",
-		"Half-Elf",
-		"Dwarf",
-		"Tiefling",
-		"Dark Elf",
-		"Aasimar"
+	allowed_sexes = list("male", "female")
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm:3]:
```suggestion
	allowed_sexes = list(MALE, FEMALE)
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

**Review by ThePainkiller — APPROVED** [review]

**Review by Stutternov — APPROVED** [review]

**Pull request merged**